### DISCUSSION TITLE
After deleting Kafka, operators will continue to watch

### DISCUSSION THREAD

**Discussion by lanzhiwang** [strimzi/strimzi-kafka-operator#6153] [General]
After I delete the kafka whose name is test123, the operators will still watch whether the related pods are ready. Have you ever encountered this situation?
![image](https://user-images.githubusercontent.com/7719138/148550998-ec5e7ad2-7610-4682-9e58-657030423ff9.png)

**Comment by scholzj** [comment]
This can happen if you delete it in the wrong time while it is in the middle of a reconciliation. It normally should not cause any issues, just timeout like this.
